sunpy/instr/rhessi.py

```diff
@@ -219,7 +219,55 @@ def get_obssum_filename(time_range):
 
     filenames = dbase_dat.get('filename')[index_number_start:index_number_end]
     return [posixpath.join(get_base_url(), 'metadata', 'catalog', filename + 's')
-            for filename in filenames]
+        for filename in filenames]
+
+
+def get_obssum_filename_multiple_months(time_range):
+    """
+    Download the RHESSI observing summary data from one of the RHESSI
+    servers, parses it, and returns the name of the obssumm files relevant for
+    the time range.
+
+    Parameters
+    ----------
+    time_range : str, TimeRange
+        A TimeRange or time range compatible string
+
+    Returns
+    -------
+    out : list of lists
+        Returns the filenames of the observation summary file for multiple months
+        in the time range
+
+    Examples
+    --------
+    >>> import sunpy.instr.rhessi as rhessi
+    >>> rhessi.get_obssum_filename_multiple_months(('2011/04/04', '2011/04/05'))   # doctest: +SKIP
+    ['http://soleil.i4ds.ch/hessidata/metadata/catalog/hsi_obssumm_20110404_042.fits']
+
+    .. note::
+        This API is currently limited to providing data from whole days only.
+
+    """
+    time_range = TimeRange(time_range)
+    filenames = []
+    delta = relativedelta(time_range.end, time_range.start)
```

Comment:
Is there a reason to use this function as opposed to just
```
delta = time_range.end - time_range.start
```

Reply:
Sorry. I've educated myself using [stackoverflow](https://stackoverflow.com/questions/12433233/what-is-the-difference-between-datetime-timedelta-and-dateutil-relativedelta) and now I'm wondering if `TimeRange` should be using that rather than `timeldeta` since it's so much more useful, @Cadair?